### ISSUE TITLE
Complete EPA downloads

### ISSUE THREAD

**Issue by titaniumbones** [edgi-govdata-archiving/overview#119]
With the EPA in crisis, we need to finish off our efforts to gather EPA datasets together.  

*Minimum success criterion:*  archive copies of all known EPA datasets currently available on the open web.

*Stretch goal:* recreate opendata.epa.gov and potentially other developer & user interfaces as well.

*Resources*
- [List of EPA datasets on the Open Web](https://edg.epa.gov/data.json)
- [EPA Digital Strategy Docs, with a number of other data lists](https://www.epa.gov/open/digital-strategy)

*Direct Contact*

Despite many discussions about this, it does not appear that EDGI or Archivers has yet made a systematic effort to reach out to EPA sysadmins and see if we can grab datasets that way. We have probably reached the time to start doing this. 

Tagging @b5, @dcwalk , @patcon to continue this conversation as we go forward.

**Comment by dcwalk** [comment]
Current status as of 2017-08-07:

- ~300 in archivers.space as URLs to research
- crawling from datatogether informing https://github.com/datatogether/coverage
- see #120 for discussion of how coverage is being calculated/generated

**Comment by dcwalk** [comment]
Reviewed this issue at Aug 14 archiving call:
> By Aug 29, ideal world we have a Data Together Node that resolves this
> 2 pieces:
> - Coverage Analysis (**static**)
> - Storage of Data (Doris Duke deliverables [mostly not EPA], Crawling EPA)

_Note: This is related to Doris Duke deliverables (need an issue to track that)_

TODO:
- [ ] Doris Duke in Datatogether @b5 (ticketed to #194)
- [ ] Refresh our machine-readable list of who has what (coverage repos) opportunity for peer work? (@b5 @patcon) (ticketed to #196)
- [ ] Review intersection of archives.space and datatogether.org ?? (wait on one above) (ticketed to #197
- [ ] (Fall) Re-reach out to look at Project Azimuth, Climate Mirror, IA (ticketed to #195)

Open threads:
- Quality? QA 
- What do we mean when we say coverage

**Comment by dcwalk** [comment]
Once #194, #196, #197, and #199 (maybe #195) are closed this can be resolved.

In the interim, moving to the [Fall Work Cycle](https://github.com/edgi-govdata-archiving/overview/milestone/2) milestone based on our [September 11 Archiving call](https://hackmd.io/GwFgJgRgrGCMCGBaAxgU1WRIDMFP1gA4B2FeCEAgTm3mAAZgg===?view)...as this was indicated as an ongoing and important priority.

**Comment by Frijol** [comment]
Per our new [stale issues policy](https://github.com/edgi-govdata-archiving/overview/blob/master/guidelines/project_guidelines.md#stale-issues-policy):

> This issue has been marked as stale because it has not had recent activity. It will be closed in seven days if no further activity occurs. If it should not be closed, please comment! Thank you for your contributions.

In the future, a robot will take care of this process!

**Comment by Frijol** [comment]
Per @dcwalk 's [comment above](https://github.com/edgi-govdata-archiving/overview/issues/119#issuecomment-328704864), all the issues mentioned are closed except https://github.com/edgi-govdata-archiving/overview/issues/197 and https://github.com/edgi-govdata-archiving/overview/issues/195 (which is stale).
